### PR TITLE
Add imagePullSecrets to fluent-bit for use with private registry

### PR DESCRIPTION
**Pull request by donbowman**
Other helm charts (e.g. Kibana) use image.imagePullSecrets,
model after them.

Signed-off-by: Don Bowman <db@donbowman.ca>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x ] Chart Version bumped
- [ x] Variables are documented in the README.md

### CODE REVIEW

File: stable/fluent-bit/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: fluent-bit
-version: 0.12.0
+version: 0.12.1
```
Comment (davidkarlsen) [inline stable/fluent-bit/Chart.yaml]: new feature . set 0.13.0

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/fluent-bit/Chart.yaml (hunk 1 above)
Comment (donbowman) [reply stable/fluent-bit/Chart.yaml]: done

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**